### DISCUSSION TITLE
404 error returned from files served from public folder

### DISCUSSION THREAD

**Discussion by bestuiexperience** [vercel/next.js#13460] [Help]
With Next JS 9.4.0, I am getting the following error for files served from the public folder:

error [index.js]: Path: images/xxx.svg | Error: Unable to resolve path \/images\/xxx.svg.
Is any additional configuration needed to serve up files from the public folder or is this a bug in Next JS?

**Comment by samceena** [comment]
what's your folder structure like?
Do you have a picture? and whats your code like?
How do you access it?
The default configuration should work.

Could it be a file permission issue?
<img width="242" alt="Screen Shot 2020-05-27 at 3 24 03 PM" src="https://user-images.githubusercontent.com/6410538/83063473-34067680-a02e-11ea-8871-ecbb91b68d13.png">
<img width="435" alt="Screen Shot 2020-05-27 at 3 24 10 PM" src="https://user-images.githubusercontent.com/6410538/83063474-34067680-a02e-11ea-9c5a-10c72eb14540.png">

**Comment by samceena** [comment]
https://github.com/samceena/nextjs-404-issue

**Reply by bestuiexperience** [reply]
I am using an index.js file in my application. In order tol solve the problem I had to do this. I am not sure why I had to escape all of those characters.

server.get(/\/.*\.(.*)/, (req, res) => {
    logger.debug(`files req.path: ${req.path}`);
    return handle(req, res);
  });
